tests/test_base.py

```diff
@@ -1236,3 +1238,29 @@ def fake_error(msg):
             self._ensure_queues()
             assert len(errors) == 1
             assert "not found" in errors[0]
+
+
+class TestRunnerClass(BaseTestCase):
+    def test_custom_runner_class_single_task(self):
+        task = self.tiger.delay(simple_task, runner_class=MyRunnerClass)
+        Worker(self.tiger).run(once=True)
+        assert self.conn.get('task_id') == task.id
+        self.conn.delete('task_id')
+        self._ensure_queues()
+
+    def test_customer_runner_class_batch_task(self):
```

Comment:
Probably should have a test that confirms what happens if two batch tasks have two different runners.

Reply:
in https://github.com/closeio/tasktiger/pull/175/commits/5653f299123cc1637583946ae08cc7c1259b9000